spec/view_models/admin/edit_auction_view_model_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'rails_helper'
+
+describe Admin::EditAuctionViewModel do
```

Comment:
Oh man, did we really have no tests? This is why I want to figure out the issues with coverage I guess!